libs/s25main/desktops/dskCredits.cpp

```diff
@@ -34,21 +35,33 @@
 #include <cstdlib>
 #include <numeric>
 
-/** @class dskCredits
- *
- *  Class for the Credits Desktop.
- */
-
 /// Duration for one credits page
-const unsigned PAGE_TIME = 12900;
+const auto PAGE_TIME = std::chrono::milliseconds(12900);
 /// Duration for fading between pages
-const unsigned FADING_TIME = 2000;
+const auto FADING_TIME = std::chrono::seconds(2);
+using std::chrono::duration_cast;
+
+namespace {
+enum
+{
+    ID_btBack,
```

Comment:
do we have `bt` or `btn` everywhere else?

Reply:
ID_bt